resources/admin.php

```diff
@@ -30,90 +30,100 @@
+    ["translatableTitle" => _("Authorized Site"),       "className" => "AdminLink", "id" => "AuthorizedSite"],
+    ["translatableTitle" => _("Cataloging Status"),     "className" => "AdminLink", "id" => "CatalogingStatus"],
+    ["translatableTitle" => _("Cataloging Type"),       "className" => "AdminLink", "id" => "CatalogingType"],
+    ["translatableTitle" => _("Contact Role"),          "className" => "AdminLink", "id" => "ContactRole"],
+    ["translatableTitle" => _("Currency"),              "className" => "CurrencyLink"],
+    ["translatableTitle" => _("Downtime Type"),         "className" => "AdminLink", "id" => "DowntimeType"],
+    ["translatableTitle" => _("EBSCO Kb Config"),       "className" => "EbscoKbConfigLink"],
+    ["translatableTitle" => _("External Login Type"),   "className" => "AdminLink", "id" => "ExternalLoginType"],
+    ["translatableTitle" => _("Funds"),                 "className" => "FundLink"],
+    ["translatableTitle" => _("Import Configuration"),  "className" => "ImportConfigLink"],
+    ["translatableTitle" => _("Export Configuration"),  "className" => "ExportConfigLink"],
+    ["translatableTitle" => _("License Status"),        "className" => "AdminLink", "id" => "LicenseStatus"],
+    ["translatableTitle" => _("Note Type"),             "className" => "AdminLink", "id" => "NoteType"],
+    ["translatableTitle" => _("Order Type"),            "className" => "AdminLink", "id" => "OrderType"],
+    ["translatableTitle" => _("Purchasing Site"),       "className" => "AdminLink", "id" => "PurchaseSite"],
+    ["translatableTitle" => _("Resource Format"),       "className" => "AdminLink", "id" => "ResourceFormat"],
+    ["translatableTitle" => _("Resource Type"),         "className" => "AdminLink", "id" => "ResourceType"],
+    ["translatableTitle" => _("Storage Location"),      "className" => "AdminLink", "id" => "StorageLocation"],
+    ["translatableTitle" => _("Subjects"),              "className" => "SubjectsAdminLink"],
+    ["translatableTitle" => _("User Limit"),            "className" => "AdminLink", "id" => "UserLimit"]
+  ];
+  if ($config->settings->enhancedCostHistory == 'Y'){
+    $adminMenuLinks[] = ["translatableTitle" => _("Cost Details"), "className" => "AdminLink", "id" => "CostDetails"];
+  }
+  if ($config->settings->enableAlerts == 'Y'){
+    $adminMenuLinks[] = ["translatableTitle" => _("Alert Settings"), "className" => "AlertAdminLink"];
+  }
+  if ($config->settings->organizationsModule == 'N'){
+    $adminMenuLinks[] = ["translatableTitle" => _("Organization Role"), "className" => "AdminLink", "id" => "OrganizationRole"];
+    $adminMenuLinks[] = ["translatableTitle" => _("Organizations"), "className" => "AdminLink", "id" => "Organization"];
+  }
+  // Sort the links according to their title (I do not believe that this is locale aware)
+  usort($adminMenuLinks, function ($item1, $item2) {
+    return strcmp($item1["translatableTitle"], $item2["translatableTitle"]);
```

Comment:
I don't know if strcmp is locale aware, but this is a good start anyhow. Not a blocker for me. Can be improved later if needed.